docs/adding games.md

```diff
@@ -248,7 +253,11 @@ checks, what options to offer for the player’s yaml file, and the code to init
 
 Here’s an example of what your world module can look like:  
   
-![Example world module directory open in Window's Explorer](./img/archipelago-world-directory-example.png)  
+![Example world module directory open in Window's Explorer](./img/archipelago-world-directory-example.png)
+
+The minimum requirements for a new archipelago world are the module itself (the world folder containing a file named `__init__.py`),
```

Comment:
```suggestion
The minimum requirements for a new archipelago world are the package itself (the world folder containing a file named `__init__.py`),
```

Technically a folder with `__init__.py` is a package. @Berserker66 educated me on this sometime this last week, haha.